src/harness/util.py

```diff
@@ -130,6 +130,17 @@ def writeConfig(config_filename, config):
         json.dumps(config, indent=2, sort_keys=False, separators=(',', ': ')))
 
 
+def writeDB(db_filename, data):
```

Comment:
This appears unused, please remove.

Reply:
removed and reverted to the master version